src/views/Fetchurl.vue

```diff
@@ -1,76 +1,189 @@
 <template>
-  <div v-if="!isLoading && !error">
-    <h1 class="page-fetch-url-header">Generate DataHub archive recovery URL</h1>
-      <form class="bg-transparent border-0 canvas-form" @submit.prevent="fetchUrl">
-        <v-container>
-          <v-row>
-            <v-col class="float-right" sm="3">
-              <label for="page-fetch-url-name" class="page-fetch-url-form-label">Enter GCP storage object URL:</label>
-            </v-col>
-            <v-col class="pl-0 pt-2" sm="9">
-              <v-text-field
-                id="page-fetch-url-name"
-                v-model="gs_source_url"
-                class="w-50"
-                :disabled="isCreating"
-                placeholder="gs://"
-              />
-            </v-col>
+  <div v-if="!currentUser" class="pt-2 page-fetch-url">
```

Comment:
`currentUser` should NEVER be null.  Thus, `!currentUser` will always be false. Do we want `!currentUser.isAuthenticated`